doc_admin/AdminManual.adoc

```diff
@@ -1523,76 +1522,76 @@ NOTE: An application restart is required to see the effect of any applied parame
 [#sms-properties]
 === 5.7 sms.properties
 
-Starting with Open Hospital version 1.8 a default sms.properties file is included:
+Starting with Open Hospital version 1.11 a default sms.properties file is included:
```

Comment:
I believe we agreed not to note when some feature or function started as the documentation is meant for the current version of the product.   So I would remove `Starting with Open Hospital version 1.11 a default sms.properties file is included:` and replace it with `The default sms.properties file is:`.

Reply:
I agree, but I searched the document and the notation was still there, so I just kept it for consistency.
I can change all occurrence at once.
Thanks!